Run Travis integration tests with the right Python version

mgedmin opened the pull request:
Fixes an issue @abadger mentioned in the comments of PR #12627.

mgedmin committed:
Run Travis integration tests with the right Python version

mgedmin committed:
Can we even ask for Python 2.4 on Travis?

mgedmin committed:
Revert "Can we even ask for Python 2.4 on Travis?"

Becaue we can't: https://travis-ci.org/mgedmin/ansible/jobs/84866458

This reverts commit e31e866e79db1e98547bc22b20da0e319955d806.

sivel commented on .travis.yml:
```diff
@@ -21,6 +25,6 @@ script:
 - if test x"$TOKENV" != x'py24' ; then  tox ; fi
 - if test x"$TOKENV" = x'py24' ; then python2.4 -V && python2.4 -m compileall -fq -x 'module_utils/(a10|rax|openstack|ec2|gce).py' lib/ansible/module_utils ; fi
```
I know you didn't touch these 2 lines, but since you are in here.  Looks like `TOKENV` should probably be `TOXENV`

mgedmin replied:
Oh my, how did I ever not notice!

mgedmin replied:
Ok, this works because of a similar misspelling of `TOKENV=py24` at the top, while `TOXENV=py26` and newer vars are spelled correctly.

mgedmin committed:
Typo: TOKENV -> TOXENV

This is purely cosmetic: the logic worked correctly, since tox never saw
the TOKENV environment variable.

abadger commented:
Thanks @mgedmin  and thanks @sivel !
